infrastructure/probate_users.tf

```diff
@@ -0,0 +1,24 @@
+data "azurerm_key_vault" "probate_key_vault" {
+  name = "${local.probateVaultName}"
+  resource_group_name = "${local.probateVaultName}"
+}
+
+data "azurerm_key_vault_secret" "probate_admin_username" {
```

Comment:
Have you added all values in probate vault path ?

Reply:
yes it is already there in demo,preprod and prod